docs/usage.md

```diff
@@ -191,7 +191,7 @@ Variant filtering is an optional step. You can skip it using `--skip_variantfilt
 
 ## Variant annotation
 
-The annotation of variants is performed using snpEff and VEP. The parameter to use is `--tools snpeff` or `--tools vep`. You can even run both snpEff and VEP using `--tools merge`, in this case the output VCF file will have both snpEff and VEP annotations combined.
+The annotation of variants is performed using snpEff, VEP and BCFtools annotate. The parameter to use is `--tools snpeff`, `--tools vep` or `--tools bcfann`. You can even run both snpEff and VEP using `--tools merge`, in this case the output VCF file will have both snpEff and VEP annotations combined.
```

Comment:
With the addition of `bcfann` this parameter is a bit confusing.  Does `merge` include `bcfann`?  Does not appear so.

Reply:
no, merge is only snpeff followed by vep